Projects/Chat/Sources/Views/InboxView.swift

```diff
@@ -65,12 +65,23 @@ public struct InboxView: View {
     private func getRightView(for conversation: Conversation) -> some View {
         if conversation.hasNewMessage {
             hText(L10n.chatNewMessage, style: .label)
-                .foregroundColor(hTextColor.Opaque.black)
+                .foregroundColor(hSignalColor.Blue.text)
                 .padding(.horizontal, .padding6)
                 .padding(.vertical, 3)
                 .background(
                     RoundedRectangle(cornerRadius: 6)
-                        .fill(hHighlightColor.Blue.fillTwo)
+                        .fill(hHighlightColor.Blue.fillOne)
+                )
+                .transition(.scale.combined(with: .opacity))
+                .matchedGeometryEffect(id: "rightView_\(conversation.id)", in: animationNamespace)
+        } else if conversation.isConversationOpen == false {
```

Comment:
doesn't it look cleaner to use if `!conversation.isConversationOpen` since we use that type of syntax elsewhere? 😋
Or maybe even create a new attribute `isConversationClosed`?

Reply:
I took you advice in and reworked it so now we have status and computed properties based on that status which makes things much more readable 👌

Reply:
awesome! Looks great 😍